parallel-consumer-core/src/main/java/io/confluent/parallelconsumer/internal/ConsumerManager.java

```diff
@@ -45,7 +59,7 @@ public class ConsumerManager<K, V> {
 
     ConsumerRecords<K, V> poll(Duration requestedLongPollTimeout) {
         Duration timeoutToUse = requestedLongPollTimeout;
-        ConsumerRecords<K, V> records;
+        ConsumerRecords<K, V> records = new ConsumerRecords<>(new HashMap<>());
```

Comment:
I think can just init to null avoiding reduntant hashmap / ConsumerRecords object creation and do a check on return
```suggestion
        ConsumerRecords<K, V> records = null;
```

Then:
```
Line 112: return records;
```
replaced with 
 ```
 return records != null ? records : new ConsumerRecords<>(UniMaps.of());
 ```
 and due to that change can remove statement in the catch block on  Line 108: ` records = new ConsumerRecords<>(UniMaps.of());`